src/tests/domParsers/videoUtils_tests.ts

```diff
@@ -22,11 +22,6 @@ test("videoDomainIfSupported should return undefined on a Vimeo user page", () =
 		"Vimeo user page should not be recognized as a page with a video");
 });
 
-test("videoDomainIfSupported should return undefined on a Vimeo ondemand page", () => {
```

Comment:
Oh, this isn't the test I was referring to on Slack, didn't know this one exists now. I see these video tests have a changed a bunch since I last took a hard look at them.

The code with the `TODO` I referenced isn't even in use anymore... I'll go ahead and clean up that dead code, might as well